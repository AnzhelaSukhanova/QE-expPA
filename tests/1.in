(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(assert
 (exists ((x (_ BitVec 4))) (or (bvult x y) (= x y))))
