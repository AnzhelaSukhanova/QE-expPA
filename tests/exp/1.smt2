(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(assert
 (exists ((x (_ BitVec 4))) (or (bvult (bvshl #b0010 x) y) (= (bvshl #b0010 x) y))))
