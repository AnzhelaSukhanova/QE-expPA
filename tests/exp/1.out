(let (($e1 (ite (bvult y #b1000) (ite (bvult y #b0100) (ite (bvult y #b0010) (ite (= y #b0000) #b0100 #b0000) #b0001) #b0010) #b0011))) (let (($e2 (bvshl #b0001 $e1))) (let (($e3 (bvshl #b0001 (bvadd #b0001 $e1)))) (not (and (not (= y $e3)) (not (= y $e2)) (not (bvult $e3 y)) (not (bvult $e2 y)) (not (= y #b0001)) (not (bvult #b0001 y)))))))